C/plugins/storage/postgres/init.sql

```diff
@@ -773,99 +727,96 @@ INSERT INTO foglamp.log_codes ( code, description )
-     VALUES ( 'SEND_PR_3', 'HTTP North Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "http_north", "default" : "http_north", "description" : "Python module name of the plugin to load" } } ');
+     VALUES ( 'SEND_PR_2',
+              'OMF North Statistics Plugin Configuration',
+              ' { "plugin" : { "type" : "string", "value" : "omf", "default" : "omf", "description" : "Python module name of the plugin to load" } } '
+            );
 
--- OCS north plugin configuration, north key-value pair should not be added and pick dynamically (TODO- FOGL-732)
+-- SEND_PR_3 - HTTP Plugin
 INSERT INTO foglamp.configuration ( key, description, value )
-     VALUES ( 'SEND_PR_4', 'OCS North Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "ocs", "default" : "ocs", "description" : "Python module name of the plugin to load" } } ');
-
+     VALUES ( 'SEND_PR_3',
+              'HTTP North Plugin Configuration',
+              ' { "plugin" : { "type" : "string", "value" : "http_north", "default" : "http_north", "description" : "Python module name of the plugin to load" } } '
+            );
 
--- STRMN: Streaming
---        status      : the process is on or off, it is on by default
---        time window : the time window when the process is active, always active by default (it means every second)
+-- SEND_PR_4 - OSIsoft Cloud Services plugin for readings
 INSERT INTO foglamp.configuration ( key, description, value )
-     VALUES ( 'STREAMING', 'Streaming', '{ "status" : "day", "window" : [ "always" ] }' );
+     VALUES ( 'SEND_PR_4',
+              'OCS North Plugin Configuration',
+              ' { "plugin" : { "type" : "string", "value" : "ocs", "default" : "ocs", "description" : "Python module name of the plugin to load" } } '
+            );
 
--- SYPRG: System Purge
---        retention : data retention in seconds. Default is 3 days (259200 seconds)
---        last purge: ts of the last purge call
-INSERT INTO foglamp.configuration ( key, description, value )
-     VALUES ( 'SYPURGE', 'System Purge', to_jsonb( '{ "retention" : 259200, "last purge" : "' || now() || '" }' ) );
 
--- COAP: South Microservice - CoAP Listener Plugin
---        plugin: python module to load dynamically
-INSERT INTO foglamp.configuration ( key, description, value )
-     VALUES ( 'COAP', 'CoAP South Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "coap_listen", "default" : "coap_listen", "description" : "Python module name of the plugin to load" } } ');
+-- South plugins
 
 -- POLL: South Microservice - POLL Plugin template
---        plugin: python module to load dynamically
 INSERT INTO foglamp.configuration ( key, description, value )
-     VALUES ( 'POLL', 'POLL South Plugin Configuration', ' { "plugin" : { "type" : "string", "value" : "poll_template", "default" : "poll_template", "description" : "Python module name of the plugin to load" } } ');
+     VALUES ( 'POLL',
```

Comment:
should we call it a template ?!

Reply:
It is called template, is it what you mean?